hiding-in-the-clouds_secret-management-with-openstack-barbican/heat-templates/master/cinder-encrypted.yaml

```diff
@@ -0,0 +1,166 @@
+# Example invocation: heat stack-create --poll -f cinder-encrypted.yaml cinder-encrypted
+heat_template_version: 2015-10-15
+
+# Some notes about the parameters:
+#
+# The parameters provided are the defaults for Openstack SUSE.
```

Comment:
Can you simply change the default parameters and remove this comment? As it is (this is meant for master), this will only run on Devstack anyway and I'd rather not have anyone accidently think this template will run on SOC before we have it fully on Newton and have the template tested.